code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -74,6 +74,7 @@
 	var/flash_mod =     1                    // Stun from blindness modifier.
 	var/fall_mod =      1                    // Fall damage modifier, further modified by brute damage modifier
 	var/grab_mod =      1                    // How easy it is to grab the species. Higher is harder to grab.
+	var/resist_mod =    0                    // How easy it is for the species to resist out of a grab. (It increases the percentage chance by the amount)
```

Comment:
If possible that should be changed to be a multiplier aswell (to be in line with the other modifiers)

Reply:
i will enter intense discussions in discord to not have to do this, because balancing this will be buuuuuuuuuuuuuuuuuuttssssssssss